### ISSUE TITLE
Request to add Person identifier as an identity element for K12 staff and student

### ISSUE THREAD

**Issue by Lakshminair-maine** [CEDStandards/CEDS-Elements#287]
This is for capturing needs not currently supported by the CEDS model.  Please do not send or share actual data as examples in this issue or in attachments. 

**Author(s)** Lakshmi Nair

**Authoring Organization(s)** DOE Maine

**Email address**  lakshmi.nair@maine.gov

**Use Case Title**
Provide a concise description that defines the use case.

**Use Case Description** 
List the proposed element name(s), definition(s), and/or option set(s).

**Use Case Background**
Provide information related to why these changes/additions are needed.

**Location of Element in the Domain Entity Schema**
[CEDS OSC Proposed New Element - Person Id.docx](https://github.com/CEDStandards/CEDS-Elements/files/7092620/CEDS.OSC.Proposed.New.Element.-.Person.Id.docx)

**Comment by 4pins** [comment]
Can't we just have multiple of these?

![Screen Shot 2021-09-01 at 9 19 45 AM](https://user-images.githubusercontent.com/10967355/131707318-af2043af-9c55-4a6b-8f6c-193cb7d3b349.png)

https://ceds.ed.gov/element/001071
https://ceds.ed.gov/element/001075
https://ceds.ed.gov/element/000618

Maybe some additional codes for the second two?

**Comment by AEMDuaneBrown** [comment]
Clarifying the request.  The element "Person Identifier" already exists in CEDS (https://ceds.ed.gov/CEDSElementDetails.aspx?TermId=18551) Is the goal of this request to make that element available in the Domain Entity Schema for the K12 > K12 Student > Identity > Identification location and the K12 > K12 Staff > Identity > Identification location? We would likely also include Person Identification System (https://ceds.ed.gov/CEDSElementDetails.aspx?TermId=18550) to go with that addition.

**Comment by 4pins** [comment]
Cool, thanks for the clarification.

**Comment by Lakshminair-maine** [comment]
That's correct. Current location of person id under Individualized Program is not serving our purpose hence we would like it added to the said path. Yes adding Person Identification System Will be great also.

Thanks,
Lakshmi Nair
